releasenotes/notes/fix-from-polynomial-tensor-constant-9df7f63be4a4c819.yaml

```
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    When using `from_polynomial_tensor` constructor for one of `BosonicOp`, `FermionicOp`,
+    `SpinOp`, or `VibrationalOp`, the coefficient for the constant term was a 0d Tensor object
```

```suggestion
    When using :meth:`SparseLabelOp.from_polynomial_tensor` constructor for one of :class:`.BosonicOp`, :class:`.FermionicOp`,
    :class:`.SpinOp`, or :class:`.VibrationalOp`, the coefficient for the constant term was a 0d Tensor object
```
I suggest we make these names proper links as shown above.

If we do not, we would still need to fix the verbatim markup to the RST style rather than Markdown one (i.e. use double backticks).

---

You van tell that I am new at this ;). Thank you!

---

PS: not sure if the 100 char line length limit applies to release notes, but fixed that just in case.

---

> PS: not sure if the 100 char line length limit applies to release notes, but fixed that just in case.

No, this limit only applies to Python files. But its good to keep these files legible in any case :+1: